### ISSUE TITLE
consider simplifying hplt.ScanXYZ API

### ISSUE THREAD

**Issue by sbinet** [go-hep/hbook#5]
for the moment, `hplt.ScanH1D` is like so:

``` go
hplt.ScanH2D(db, "select x, y from ntuple where z>1", h)
```

ROOT's `TTree::Draw` is like so:

``` c++
tree.Draw("x:y", "z>1", "drawing-option", nentries, firstentry);
```

perhaps consider:
- a way to hide the `select`
- a way to hide the `from ntuple-name` (should be trivial when #4 is in)

ie:

``` go
nt := hbook.NewNtuple(...)

// 1D selection
err = nt.ScanH1D("x", h)
err = nt.ScanH1D("x where z>1", h)

// 1D selection, taking the entries from [10,100)
err = nt.ScanH1D("x where (z>1 && id() >= 10 && id() < 100)", h)

// 2D selection
err = nt.ScanH2D("x,y where z>1", h2)
```

**Comment by sbinet** [comment]
@ebusato

**Comment by sbinet** [comment]
fixed with bd2bcd7c04e23d01e461aa3bcba17a57d3e651a4
